fsharp-backend/paket.lock

```diff
@@ -818,33 +818,33 @@ NUGET
     System.Windows.Extensions (5.0)
       System.Drawing.Common (>= 5.0)
 
-GROUP Wasm
+GROUP Analysis
 NUGET
   remote: https://api.nuget.org/v3/index.json
-    Microsoft.AspNetCore.Authorization (6.0.1) - restriction: >= net6.0
-      Microsoft.AspNetCore.Metadata (>= 6.0.1) - restriction: || (>= net461) (>= netstandard2.0)
-      Microsoft.Extensions.Logging.Abstractions (>= 6.0) - restriction: || (>= net461) (>= netstandard2.0)
+    Microsoft.AspNetCore.Authorization (6.0.5) - restriction: >= net6.0
```

Comment:
There's a lot of version changes here. How come and what should we do about them?

Reply:
I think paket just refreshed some versions when I changed the group name from `wasm` to `analysis`.

Personally I wasn't concerned about them - they should be non-breaking changes, and we should have tests covering these things.

Reply:
Cool, thanks for clarifying.